templates/emails/customer-processing-order.php

```diff
@@ -27,7 +27,7 @@
 <?php /* translators: %s: Customer first name */ ?>
 <p><?php printf( esc_html__( 'Hi %s,', 'woocommerce' ), esc_html( $order->get_billing_first_name() ) ); ?></p>
 <?php /* translators: %s: Order number */ ?>
-<p><?php printf( esc_html__( 'Just to let you know &mdash; your payment has been confirmed, and order #%s is now being processed:', 'woocommerce' ), esc_html( $order->get_order_number() ) ); ?></p>
+<p><?php printf( esc_html__( 'Just to let you know &mdash; your order, order #%s has been confirmed, and is now being processed:', 'woocommerce' ), esc_html( $order->get_order_number() ) ); ?></p>
```

Comment:
I'd be inclined to avoid "confirmed" as that's a status that comes after the processing is done. How about 

`'Just to let you know &mdash; we've received your order #%s, and it is now being processed'`

Reply:
Obviously ditto for the other plain text template.